src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenRefReturnTests.vb

```diff
@@ -1080,7 +1080,7 @@ End Module",
         ''' <summary>
         ''' Late-bound calls with ByRef return values not supported.
         ''' </summary>
-        <Fact()>
+        <ConditionalFact(GetType(IsEnglishLocal))>
```

Comment:
><ConditionalFact(GetType(IsEnglishLocal))> [](start = 8, length = 42)

Instead of doing this, we can change/restore CurrentCulture in the Main method of comp2. We should have examples of that.